### PR TITLE
Replace SmartR R scripts dir config with Heim own folder. Trim down SmartR specific Frontend parts.

### PR DESCRIPTION
**Pull request by PiotrZakrzewski**

### CODE REVIEW

**Comment by PiotrZakrzewski** [comment]
R scripts were removed so that they do not show up in the script list.

**Comment by PiotrZakrzewski** [comment]
@forus Can you have another look?

**Comment by forus** [comment]
@PiotrZakrzewski It looks like there is another place that points to `Scripts` folder: https://github.com/thehyve/heim-SmartR/blob/e5b91637b4be2f7108faf38f0189d82d470aada5/grails-app/services/smartR/plugin/SmartRService.groovy#L100
Wouldn't it be an idea to introduce a method in the service to return the path to the scripts. Something like `SmartRService.getScriptsFolder()`

**Comment by PiotrZakrzewski** [comment]
Done. 
There is an issue with Build (when running the plugin standalone) however so I could not run unit-test for it.

File: grails-app/services/smartR/plugin/SmartRService.groovy [hunk 1]
```diff
@@ -116,4 +117,11 @@ class SmartRService {
 
         return scriptExecutorService.run(parameterMap)
     }
+
+
+    def getScriptsFolder(){
+        def scriptsFolderName = "HeimScripts"
+        def scriptFolder = "/${scriptsFolderName}/".toString() //In order to pass Java String instead of GString
+        return getWebAppFolder() +scriptFolder
+    }
```
Comment (forus) [inline grails-app/services/smartR/plugin/SmartRService.groovy:125]: GString should be converted to the String when you will declare the return type of the method explicitly `String getScriptsFolder()`
Then you could return simply "${webAppFolder}/${scriptsFolderName}/"

File: grails-app/controllers/smartR/plugin/SmartRController.groovy [hunk 2]
```diff
@@ -15,7 +15,7 @@ class SmartRController {
     *   Renders the default view
     */
     def index = {
-        def dir = smartRService.getWebAppFolder() + '/Scripts/'
+        def dir = smartRService.getWebAppFolder() + '/HeimScripts/'
```
Comment (forus) [inline grails-app/controllers/smartR/plugin/SmartRController.groovy]: Please reuse `getScriptsFolder()` method.

**Pull request merged**